app/services/batched_remove_status_service.rb

```diff
@@ -57,22 +51,6 @@ def batch_stream_entries(account, statuses)
     end
   end
 
-  def batch_activity_json(account, statuses)
-    account.followers.inboxes.each do |inbox_url|
-      statuses.each do |status|
-        @activity_json_batches << [build_json(status), account.id, inbox_url]
-      end
-    end
-
-    statuses.each do |status|
-      other_recipients = (status.mentions + status.reblogs).map(&:account).reject(&:local?).select(&:activitypub?).uniq(&:id)
```

Comment:
`other_recipients` is needed to handle non-followers, but RawDistributionWorker doesn't handle those recipients...?

Reply:
Also we need to handle reblogs even for Delete -> Actor from remote instance. This has been handled in Delete -> Note before

https://github.com/tootsuite/mastodon/blob/7571c37c991c7bcaf83c524105f4a3b64b19670c/app/lib/activitypub/activity/delete.rb#L31-L40

but now we only relays Delete -> Actor.